app/jsx/components/NotYetLink.jsx

```diff
@@ -0,0 +1,33 @@
+// ##### NotYetLink Component (temporary modal for for functionality/pages that are "Coming Soon") ##### //
+
+import React from 'react'
+import ModalComp from '../components/ModalComp.jsx'
+
+class NotYetLink extends React.Component {
+  state = { modalOpen: false }
+
+  closeModal = e => {
+    this.setState({modalOpen:false})
+  }
+
+  content = (<div>Ahem...This page or functionality is not yet working.<br/>We appreciate your patience.</div>)
+
+  render() {
+    return (
+      <div id="notYetLinkModalBase">
```

Comment:
I think this id and the parentSelector below aren't necessary for this component, and will in fact break things if there are multiple NotYetLinks on a page. The only reason we messed around with those for the Department modal was because we needed it to be nested within the form. I think react-modal's default behavior (that is, without parentSelector) will probably work fine for NotYetLink.